containers/ecr-viewer/src/app/services/listEcrDataService.ts

```diff
@@ -245,6 +247,31 @@ export const getTotalEcrCount = async (
   return number.count;
 };
 
+const getTotalEcrCountSqlServer = async (
+  searchTerm?: string,
+  filterConditions?: string[],
+): Promise<number> => {
+  let pool = await get_pool();
+
+  try {
+    const whereStatement = generateWhereStatementSqlServer(
+      searchTerm,
+      filterConditions,
+    );
+
+    let query = `SELECT COUNT(DISTINCT ed.eICR_ID) FROM ecr_data ed LEFT JOIN ecr_rr_conditions erc ON ed.eICR_ID = erc.eICR_ID WHERE ${whereStatement}`;
+
+    const count = await pool.request().query<number>(query);
+
+    return count.recordset[0];
```

Comment:
The eCR summary page is broken because the result of this query is actually an object containing the record count.
```suggestion
        let query = `SELECT COUNT(DISTINCT ed.eICR_ID) as count FROM ecr_data ed LEFT JOIN ecr_rr_conditions erc ON ed.eICR_ID = erc.eICR_ID WHERE ${whereStatement}`;

    // It might be better to create an interface for this return type at the top of the file
    const count = await pool.request().query<{ count: number }>(query);

    return count.recordset[0].count;
```